src/Value.ml

```diff
@@ -8,7 +8,7 @@ module T = struct
          | Char of char
          | String of string
          | List of t list
-         | BitVec of int*int
+         | BitVec of int*string
```

Comment:
A `string` representation for bit vectors is not ideal ... To store 8 bits you would end up using at least 8 bytes = 64 bits.

Take a look at the `BitArray` module in [Bitarray.ml(https://github.com/SaswatPadhi/LoopInvGen/blob/master/src/Bitarray.ml) file -- it "packs" bits together and stores (62 of) them inside a single integer.

Reply:
Concretely, use `BitVec of int * Bitarray.t`.

Reply:
Do you recommend that I add functionality directly to the Bitarray.ml file by adding all the operations like compare, shift, multiply, and add to the Bitarray type? I would then use these operations in the BitvectorComponents.ml file and then create a new logic in Logic.ml.

Reply:
Exactly 👍